AElf.Kernel/AccountManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AElf.Kernel
+{
+    class AccountManager : IAccountManager
+    {
+        public List<IAccount> ExecuteTransactionAsync(ITransaction tx)
+        {
+            var accountFrom = tx.AccountFrom;
+            var accountTo = tx.AccountFrom;
+            var amount = tx.Amount;
```

Comment:
There should not exist a amount field, a transaction is just a method call message such as 

"methodname","param1","param2"